validation/glean/glean.1.metrics.pass.json

```diff
@@ -90,6 +90,9 @@
     "url": {
       "examples.url_example": "https://mozilla.org/?x=%D1%88%D0%B5%D0%BB%D0%BB%D1%8B"
     },
+    "text": {
+      "examples.text_example": "let's goooooooooooooooooo\nmore text to follow here."
```

Comment:
I would make this at least 100 characters long to test the new limits.

Reply:
A failing example might also be a good idea.

Reply:
a failing one will be ... hm, I could generate a bunch of characters, sure.
(Not even [Alice in Wonderland](https://www.gutenberg.org/files/11/11-0.txt) is long enough on its own)